LevelUp.Pos.ProposedOrders.Tests/CalculatorTests.cs

```diff
@@ -60,25 +60,5 @@ private void RunTestArray(int[,] values)
                     i, expectedCheckValues, actualCheckValues, Environment.NewLine);
             }
         }
-
-        [TestMethod]
-        public void TestTest123()
```

Comment:
Did this test ever have any value?  Does it test anything not covered by the "test battery" stuff?